view/frontend/web/js/instantsearch.js

```diff
@@ -19,596 +19,701 @@ define([
 ], function (Component, $, algoliasearch, instantsearch, algoliaCommon, algoliaBase64, templateEngine, priceUtils) {
 
     return Component.extend({
-        initialize(config, element) {
-            // console.log('IS initialized with', config, element);
-            this.buildInstantSearch();
+        initialize(_config, _element) {
+            this.buildInstantSearch().then(() => console.log("[Algolia] InstantSearch build complete"));
         },
 
         isStarted: false,
 
+        minQuerySuggestions: 4,
```

Comment:
Is there a particular reason why this is the only value that is needed here ?

Reply:
It was hardcoded in the old version so I lifted the value in case someone wanted to write a simple mixin to override it without having to copy and hack the entire file. We could add an admin config later if we wish but I don't think it's that critical.